client/my-sites/theme/theme-support-tab/index.jsx

```diff
@@ -77,32 +82,59 @@ export default function ThemeSupportTab( { themeId } ) {
 						</Button>
 					</Card>
 
-					<Card className="theme__sheet-card-support">
-						<Gridicon icon="comment" size={ 48 } />
-						<div className="theme__sheet-card-support-details">
-							{ translate( 'Contact support' ) }
-							<small>
-								{ translate(
-									'Get answers from our AI assistant, with access to 24/7 expert human support on paid plans.'
-								) }
-							</small>
-						</div>
-						<Button
-							__next40pxDefaultSize
-							onClick={ () => {
-								setNavigateToOdie();
-								dispatch(
-									recordTracksEvent( 'calypso_theme_sheet_button_click', {
-										theme_name: themeId,
-										button_context: 'help-center-ai',
-									} )
-								);
-							} }
-							variant="secondary"
-						>
-							{ translate( 'Get in touch' ) }
-						</Button>
-					</Card>
+					{ themeTier?.slug === 'community' || themeTier?.slug === 'partner' ? (
+						<Card className="theme__sheet-card-support">
+							<Gridicon icon="notice-outline" size={ 48 } />
+							<div className="theme__sheet-card-support-details">
+								{ translate( 'Help and support for this theme is not offered by WordPress.com.' ) }
+								<small>
+									{ translate( 'Contact the theme developer directly for help with this theme.' ) }
+								</small>
+							</div>
+							<Button
+								__next40pxDefaultSize
+								onClick={ () => {
+									openSupportDoc();
+									dispatch(
+										recordTracksEvent( 'calypso_theme_sheet_button_click', {
+											theme_name: themeId,
+											button_context: 'themes-unsupported',
+										} )
+									);
+								} }
+								variant="secondary"
+							>
+								{ translate( 'Learn more' ) }
+							</Button>
+						</Card>
+					) : (
+						<Card className="theme__sheet-card-support">
+							<Gridicon icon="comment" size={ 48 } />
+							<div className="theme__sheet-card-support-details">
+								{ translate( 'Contact support' ) }
+								<small>
+									{ translate(
+										'Get answers from our AI assistant, with access to 24/7 expert human support on paid plans.'
+									) }
+								</small>
+							</div>
+							<Button
+								__next40pxDefaultSize
+								onClick={ () => {
+									setNavigateToOdie();
```

Comment:
If you want to start a new chat, you're going to need to use `await resetSupportInteraction();`